client/src/components/DataFiles/DataFilesModals/DataFilesUploadModal.js

```diff
@@ -21,18 +21,24 @@ export const LAYOUTS = ['', ...Object.keys(LAYOUT_CLASS_MAP)];
 const DataFilesUploadModal = ({ className, layout }) => {
   const history = useHistory();
   const location = useLocation();
+  const [disabled, setDisabled] = useState(false);
+
+  const [uploadedFiles, setUploadedFiles] = useState([]);
+
   const reloadCallback = () => {
     history.push(location.pathname);
+    const errorFiles = uploadedFiles.filter(f => status[f.id] === 'ERROR');
+    errorFiles.length === uploadedFiles.length && setDisabled(false);
```

Comment:
is `errorFiles` used?

Reply:
Yes. Using if sentence to make code more readable.